core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -663,8 +663,7 @@ public void changeCacheFlag(int columnIndex, boolean cache) {
         } else {
             return;
         }
-
-        txWriter.bumpStructureVersion(this.denseSymbolMapWriters);
+        updateMetaStructureVersion();
```

Comment:
Updating symbol cache flag done directly in symbols and not _meta file. This will make _meta change to match _txn structure version.